pkg/ui/console/client.go

```diff
@@ -193,10 +193,12 @@ func (c Client) DownloadProviders(ctx context.Context) (diags diag.Diagnostics)
 	return diags
 }
 
-func (c Client) Fetch(ctx context.Context) (*core.FetchResponse, diag.Diagnostics) {
+func (c Client) Fetch(ctx context.Context) (result *core.FetchResponse, diags diag.Diagnostics) {
 	if _, dd := c.SyncProviders(ctx, c.cfg.Providers.Names()...); dd.HasErrors() {
 		return nil, dd
 	}
+	defer printDiagnostics("Fetch", &diags, viper.GetBool("redact-diags"), viper.GetBool("verbose"))
+
```

Comment:
you already have printDiagnostics below, so one of the two neds to be used, otherwise it will print twice

Reply:
Oh. you are right. In my case it was printing only once so I didn't notice it.